packages/nextjs/hooks/flashbotRecoveryBundle/useRecoveryProcess.ts

```diff
@@ -44,9 +46,11 @@ export const useRecoveryProcess = () => {
   useEffect(() => {
     (async () => {
       if (!targetNetwork || !targetNetwork.blockExplorers) return;
+      const provider = new ethers.providers.InfuraProvider(targetNetwork.id);
+      setInfuraProvider(provider);
```

Comment:
infuraProvider = new ethers.providers.InfuraProvider(targetNetwork.id); and remove the setInfuraProvider